components/StyledFilters.js

```diff
@@ -10,13 +10,15 @@ const FilterButton = styled(StyledButton).attrs({
   buttonSize: 'tiny',
   buttonStyle: 'standard',
 })`
+  font-size: 14px;
```

Comment:
Just to note, these changes will effect all the `StyledFilters`. We are currently discussing this whether this is the intended behaviour; https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP?node-id=16237:237513#193987564.

Reply:
And as per the discussion above indeed needs to be applied globally; so this change stands. 🙂